plotter/image.go

```diff
@@ -13,6 +13,16 @@ import (
 	"gonum.org/v1/plot/vg/draw"
 )
 
+// uniform is a cropped uniform image.
```

Comment:
I'd usually put helpers after their use; this isn't a focus, just an incidental item.